numpy/lib/twodim_base.py

```diff
@@ -32,6 +32,76 @@ def _min_int(low, high):
     return int64
 
 
+def flip(m, axis):
```

Comment:
Since this is a function that works on arbitrary dimensions, not just 2-d, is this the right file to put it? Perhaps `function_base.py` and `tests/test_function_base.py` would be a better choice?